src/client/styles/style-v2.scss

```diff
@@ -1,3 +1,15 @@
+@import "nypl-normalize";
+@import "toolkit";
+@import "header";
```

Comment:
You are right, just did a quick copy-paste from the toolkit code since that's how we are rollin' at the moment...